.buildkite/commands/release-upload.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+
+"$(dirname "${BASH_SOURCE[0]}")/shared_setup.sh"
```

Comment:
The first test build, https://buildkite.com/automattic/pocket-casts-ios/builds/8883#01934715-f8b2-453f-8e5f-7dbda4095e71, showed that this is inefficient — which I should have realized myself — because it downloads pods and Swift Packages that we don't need in this step:

![image](https://github.com/user-attachments/assets/aa4320e8-a243-4016-98b1-263350d2df17)